apis/capabilities/v1alpha1/tenant_types.go

```diff
@@ -40,6 +40,12 @@ type TenantSpec struct {
 	TenantSecretRef        v1.SecretReference `json:"tenantSecretRef"`
 	PasswordCredentialsRef v1.SecretReference `json:"passwordCredentialsRef"`
 	MasterCredentialsRef   v1.SecretReference `json:"masterCredentialsRef"`
+	// additional parameters, used for Update, as in master portal Api Docs
+	FromEmail           string `json:"fromEmail"`
```

Comment:
If they are optional fields, better to have `*string` type. Json tag should have `omitempty` property.

Reply:
These fields should be used as well on tenant creation if possible (i.e. if the 3scale API allows)

Reply:
Thank you for comments!  Optional fields - updated. Tenant creation  , as I see, does not have option for optional params - https://github.com/3scale/3scale-porta-go-client/blob/main/client/tenants.go#L17. Thank you